pkgs/applications/science/medicine/aliza/default.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, wrapQtAppsHook
+, itk
+, cmake
+, qtbase
+}:
+
+stdenv.mkDerivation rec {
+  pname = "aliza";
+  version = "1.9.7";
+
+  src = fetchFromGitHub {
+    owner = "AlizaMedicalImaging";
+    repo = "AlizaMS";
+    rev = "v${version}";
+    hash = "sha256-bambl3NP42KEwh6NPb8jhQlGqaUCSt6QuzUmmBnEAQw=";
+  };
+
+  nativeBuildInputs = [
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    itk
+    cmake
+    qtbase
+  ];
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE:STRING=Release"
+    "-DALIZA_QT_VERSION:STRING=5"
+    "-DITK_DIR:STRING=${itk}"
+    "-DMDCM_USE_SYSTEM_ZLIB:BOOL=ON"
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm555 ./bin/* -t $out/bin
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Aliza MS DICOM Viewer";
```

```suggestion
    description = "A DICOM Viewer";
```

Do not be self-referent on description (except in cases of inside-jokes like  "GNU's Not Unix").